apollo-api/src/main/java/com/apollostack/api/Query.java

```diff
@@ -1,17 +1,34 @@
 package com.apollostack.api;
 
-import java.util.List;
+import java.util.Collections;
 import java.util.Map;
 
 /** TODO */
-public interface Query {
+public interface Query<V extends Query.Variables> {
```

Comment:
ok fair enough, I haven't used GraphQL in production yet to know how often you use variables, but sounds like it's something quite useful